<a href="https://colab.research.google.com/github/JaydaBubel/lab-inferential-statistics/blob/master/lab_inferential_stats.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Step 1: Define the null hypothesis - This is our assumption about the population. It is defined by H0 and in this case H0: **μ = 120 mm Hg;**

Step 2: Define the alternative hypothesis - This means, what if our assumption is not true. It is defined by Ha and in this case Ha: μ > 120.

Step 3: Determine if it is a one-tailed or a two-tailed test. In this case **it would be two-tailed**, because two-tailed is when the mean tested (alternative hypothesis) can be > or < then the mean of the population. Whether higher or lower, we want to see how it varies from general population on mainland.


Step 4: Decide a test statistics based on the information available; we will try a t-test.


Step 6: Calculate the test statistic based on the given information, first by hand then python.

In [1]:
import math
sample_mean = 130.1
population_mean = 120
sample_std = 21.21
sample_size = 100
t_statistic = (sample_mean - population_mean) / (sample_std / math.sqrt(sample_size))
print("Calculated Test Statistic:", t_statistic)

Calculated Test Statistic: 4.761904761904759


Step 7: Check the table to determine the critical value.
For z-test you have fixed values according to Confidence Level.
For t-test you have to calculate according to the degrees of freedom (df), which is the sample_size - 1.

In [2]:
from scipy.stats import t
alpha = 0.05
df = sample_size - 1

#Calculate critical t-val
critical_t_value = t.ppf(1 - alpha/2, df)
print("Critical t-value:", critical_t_value)

Critical t-value: 1.9842169515086827


Step 8: Make conclusions:
If the test statistic falls in the critical region, then we reject the Null Hypothesis. If the test statistic falls in the region between the critical region, then we fail to reject the Null Hypothesis.

Since the calculated test statistic (4.76) is greater than the critical t-value (1.98), **i'd reject the null hypothesis**. The results shows sig. difference between sample gourp (Honolulu) and general population - the difference here is too great to just be considered chance and indicates that the Honolulu group has higher blood pressure on avg.